app/grants/templates/grants/explorer.html

```diff
@@ -91,6 +91,7 @@
         </div>
       {% endif %}
       <div class="container mt-1">
+        <a href="javascript:history.go(-3)" class="font-body pe-auto py-2 d-inline-block"><i class="fas fa-chevron-left mr-2"></i>Back </a>
```

Comment:
-3 is here instead of -1 because it looks like JS considers adding the query parameters as a navigation event. So -1 was just changing the query params

Reply:
What happens if a user navigates directly to https://gitcoin.co/grants/explorer and then clicks the back button?